lib/render/user.php

```diff
@@ -220,11 +220,10 @@ function RenderSiteAwards($userAwards)
                 }
 
                 $linkdest = ''; //    TBD: referrals page?
-            } elseif ($awardType == 5) { //    Signed up for facebook!
-                $tooltip = "Awarded for associating their account with Facebook! Thanks for spreading the word!";
-
-                $imagepath = "/Images/_FBAssoc.png";
-                $linkdest = "/controlpanel.php#facebook";
+            // } elseif ($awardType == 5) { // Signed up for facebook!
+            //     $tooltip = "Awarded for associating their account with Facebook! Thanks for spreading the word!";
+            //     $imagepath = "/Images/_FBAssoc.png";
+            //     $linkdest = "/controlpanel.php#facebook";
```

Comment:
why comment out instead of remove?

Reply:
I kept those for documentation purposes.`SiteAward` entries of that type still exist. 

Usually i'd remove dead code but this is the only reference to the magic number `5` - there are no DB comments on the respective column.

I went ahead and introduced the AwardType class which does the documentation job instead and removed those blocks.

Reply:
👍